src/main/asciidoc/java/index.adoc

```diff
@@ -254,9 +246,6 @@ Then override the default JGroups config so that `KUBE_PING` becomes the discove
   <MFC max_credits="2m"
 *        min_threshold="0.40"
 *   />
-  <FRAG2/>
```

Comment:
Without FRAG the largest message you can send is the maximum amount of credits which above is only 2m.